plugins/quay-actions/dist-dynamic/package.json

```
@@ -44,8 +44,8 @@
   ],
   "bundleDependencies": true,
   "peerDependencies": {
-    "@backstage/plugin-scaffolder-node": "^0.2.6",
-    "@backstage/backend-plugin-api": "^0.6.6",
+    "@backstage/plugin-scaffolder-node": "^0.2.10",
+    "@backstage/backend-plugin-api": "^0.6.9",
     "@backstage/backend-plugin-manager": "npm:@janus-idp/backend-plugin-manager@0.0.2-janus.5"
```

```suggestion
    "@backstage/backend-dynamic-feature-service": "0.1.0"
```

Of course the import in the code should be changed as well.
This package is now part of the upstream backstage since Backstage 1.22.0.